source/_integrations/autoskope.markdown

```diff
@@ -0,0 +1,179 @@
+| `gps_accuracy`     | Estimated accuracy of the GPS signal in meters.                             | `3.0`                          |
+| `battery_voltage`  | Voltage of the device's internal battery (if available).                    | `4.1`                          |
+| `external_voltage` | Voltage of the vehicle's main battery connected to the device (if available). | `12.5`                         |
+| `gps_quality`      | Raw GPS quality indicator from the API (used to calculate `gps_accuracy`).  | `1.5`                          |
+| `imei`             | Unique IMEI number of the tracking device.                                  | `123456789012345`              |
+| `model`            | Model name/type of the tracking device.                                     | `Autoskope V3`                 |
+| `speed`            | Current speed of the vehicle in the unit system configured in Home Assistant. | `55.0`                         |
+| `park_mode`        | Boolean indicating if the device reports being in park mode.                | `true` / `false`               |
+| `last_update`      | Timestamp of the last position update received from the API (UTC).          | `2025-01-01T10:05:00Z`         |
+| `activity`         | Current activity state derived from `park_mode` and `speed`.                | `parked` / `moving`            |
+
+Note: Sensor entities for battery voltages and other values are planned for a future update.
+
+## Data update
+
+The integration {% term polling polls %} the Autoskope API every 60 seconds to retrieve the latest status and location data for all registered vehicles associated with the account.
+
+## Troubleshooting
+
+* **Authentication Errors:** If you change your Autoskope password, the integration will fail to connect. Home Assistant should prompt you to re-authenticate. Go to **Settings** > **Devices & Services**, find the Autoskope integration card, and click **Reconfigure** to enter your new password.
+* **Connection Failed during Setup:** Ensure you have entered the correct **Host** URL for the Autoskope API. The default value (`https://portal.autoskope.de`) should be correct for most users. Also, verify your username and password. Check if your Home Assistant instance can reach the Autoskope host URL (e.g., firewall rules, DNS issues).
+* **Vehicles Not Appearing:** If a vehicle you expect to see is missing in Home Assistant, log in to the Autoskope web portal or app and confirm that the vehicle is correctly registered, active, and associated with the account you used to set up the integration.
+* **Entities Become Unavailable:** If entities become unavailable, check the Home Assistant logs (**Settings** > **System** > **Logs**) for errors related to the Autoskope integration. This could indicate temporary connectivity issues with the Autoskope API or problems with your account credentials.
```

Comment:
```suggestion
* **Entities Become Unavailable:** If entities become unavailable, check the Home Assistant logs {% my logs title="**Settings** > **System** > **Logs**" %} for errors related to the Autoskope integration. This could indicate temporary connectivity issues with the Autoskope API or problems with your account credentials.
```
we have a my link for this :smile: